testdata/sample-config.json

```diff
@@ -10,5 +10,6 @@
     "aws_access_key": "1",
     "aws_secret_access_key": "2",
     "sink": "file",
+    "version": "1.0.0",
```

Comment:
Version is never passed in a configuration. It's a property of the software. Example:

Android 4.4 cannot pretent to be Android 7.0 by changing a configuration value.